<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPrefGeneralBase</class>
 <widget class="QWidget" name="FPrefGeneralBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Comportamiento</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="ChBMaximizado">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Inicia maximizada la aplicacion.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Inicia la aplicacion cada vez con la ventana maximizada, ocupando todo el escritorio.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Iniciar ventana principal maximizada</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ChBSplash">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Mantener pantalla de inicio al iniciar el programa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ChBIconoBandeja">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Usar icono en la bandeja del sistema</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBReloj">
        <property name="text">
         <string>Mostrar reloj digital</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Estilo y Colores</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Estilo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBEstilo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Cambia el estilo visual.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Permite permutar entre los distintos estilos disponibles para la aplicacion. Para ello, elija uno de la lista desplegable y presione el boton &quot;Aplicar&quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="CkBSobreEstilo">
          <property name="text">
           <string>SobreEstilo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBSobreEstilo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GBActualizaciones">
     <property name="title">
      <string>Servidor de Actualizaciones</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Servidor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LEServidor"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Puerto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBPuerto"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ruta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LERuta"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="LUsuario">
          <property name="text">
           <string>Usuario:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEUsuario"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="LContra">
          <property name="text">
           <string>Contrase√±a:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEContra"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBAnonimo">
        <property name="text">
         <string>Anonimo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CkBSobreEstilo</sender>
   <signal>toggled(bool)</signal>
   <receiver>CBSobreEstilo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
